beacon_node/lighthouse_network/src/peer_manager/peerdb/peer_info.rs

```diff
@@ -22,6 +22,9 @@ use PeerConnectionStatus::*;
 pub struct PeerInfo<T: EthSpec> {
     /// The peers reputation
     score: Score,
+    /// Time at which peer was banned
+    #[serde(skip)]
+    banned_since: Option<Instant>,
```

Comment:
I'm not sure this is the best way to keep track of this. Banned peers exist outside the `PeerInfo`, this keeps the posibility of inconsistency between that tracking and the inner one